### PR TITLE
Cryoemio ctf

### PR DESCRIPTION
**Pull request by arjunsingh3600**
changes to io functions for simSPI to add support for ctf parameters

### CODE REVIEW

**Comment by jedyeo** [comment]
This is ready for review but this is a tricky merge conflict :( @geoffwoollard

**Comment by ninamiolane** [comment]
The merge conflict is happening because this file is not called cryoemio anymore? Maybe use the correct new name for this file and try again?

Looking good otherwise.

**Review by ninamiolane — APPROVED** [review]
LGTM if conflict gets solved!

**Review by geoffwoollard — APPROVED** [review]
So what's the reason for the merge conflict and what's the plan to resolve?

**Comment by jedyeo** [comment]
@ninamiolane are we good to merge this?

**Review by ninamiolane — CHANGES_REQUESTED** [review]
The merge with master has failed, some files with the old version are in there...

**Comment by jedyeo** [comment]
@ninamiolane i think i fixed it, but don't want to merge without your approval :')

**Comment by jedyeo** [comment]
@ninamiolane think everything checks out!

File: ioSPI/cryo.py
Comment (ninamiolane) [inline ioSPI/cryo.py:381]: This should respect the convention for ioSPI (discussed in the PR #46 just merged)

- first argument is where we will write, and it is called "path"
- second argument is what we will write

File: ioSPI/cryo.py
Comment (ninamiolane) [inline ioSPI/cryo.py:386]: Adapt docstring

File: tests/test_atomic_models.py
Comment (ninamiolane) [inline tests/test_atomic_models.py:52]: Nice! 🎉

**Review by ninamiolane — CHANGES_REQUESTED** [review]
Thanks for this! A few things to adapt:
- there has been some GitHub issue, as atomic_models.ipynb, save.py, ml_cryo.py should have been removed (as per my review of the codebase)
- please respect the convention decided on ioSPI for the read/write functions

**Comment by geoffwoollard** [comment]
We're going ahead with this PR? It has tem simulator specifics in it? The discussion at https://github.com/compSPI/ioSPI/pull/25 suggested that this would be moved to simSPI.

**Comment by jedyeo** [comment]
> We're going ahead with this PR? It has tem simulator specifics in it? The discussion at #25 suggested that this would be moved to simSPI.

See @thisTyler 's comment at https://github.com/compSPI/ioSPI/pull/25#issuecomment-1040634866

**Review by ninamiolane — CHANGES_REQUESTED** [review]
I don't understand this PR: 
- it is almost only bringing back files that should be deleted (see previous PRs): is this a GitHub error?
- it is only dealing with imports (..ioSPI): why the change? and that does not match the title of the PR...? Nothing on the CTF?

File: tests/test_fourier.py
Comment (geoffwoollard) [inline tests/test_fourier.py:5]: fourier ins't in ioSPI anymore.

**Comment by geoffwoollard** [comment]
> I don't understand this PR:
> 
> * it is almost only bringing back files that should be deleted (see previous PRs): is this a GitHub error?
> * it is only dealing with imports (..ioSPI): why the change? and that does not match the title of the PR...? Nothing on the CTF?

Those are all good points @ninamiolane - I am confused too. @jedyeo is Nina right? Or perhaps something was lost in a commit?

Also, `fourier` isn't in` ioSPI `anymore.... this wouldn't pass tests anymore. See https://github.com/compSPI/ioSPI/pull/49

**Comment by jedyeo** [comment]
@ninamiolane @geoffwoollard after discussing with @arjunsingh3600  it seems that #46 has made this PR redundant (i.e. the refactoring that took place pulled out many of the changes that we made in this branch), but before I'm comfortable with closing this, I want to touch base with @thisTyler early tomorrow and just see that we're all on the same page.

**Comment by thisTyler** [comment]
Closing this PR as it has been messed up by refactoring quite significantly - we will remake these changes in a fresh branch.

**Pull request closed without merging**
